meson.build

```diff
@@ -1,20 +1,25 @@
+
 project('webp-pixbuf-loader', 'c')
+add_project_arguments('-pthread','-I/usr/include/gtk-3.0','-I/usr/include/at-spi2-atk/2.0','-I/usr/include/at-spi-2.0','-I/usr/include/dbus-1.0','-I/usr/lib/x86_64-linux-gnu/dbus-1.0/include','-I/usr/include/gtk-3.0','-I/usr/include/gio-unix-2.0/','-I/usr/include/cairo','-I/usr/include/pango-1.0','-I/usr/include/harfbuzz','-I/usr/include/pango-1.0','-I/usr/include/atk-1.0','-I/usr/include/cairo','-I/usr/include/pixman-1','-I/usr/include/freetype2','-I/usr/include/libpng16','-I/usr/include/freetype2','-I/usr/include/libpng16','-I/usr/include/gdk-pixbuf-2.0','-I/usr/include/libpng16','-I/usr/include/glib-2.0','-I/usr/lib/x86_64-linux-gnu/glib-2.0/include', language : 'c')
+add_project_link_arguments('-lgtk-3','-lgdk-3','-lpangocairo-1.0','-lpango-1.0','-latk-1.0','-lcairo-gobject','-lcairo','-lgdk_pixbuf-2.0','-lgio-2.0','-lgobject-2.0','-lglib-2.0', language : 'c')
```

Comment:
Okay, so, you should try to avoid hardcoding flags like this.

We use pkg-config to avoid this and make things dynamic across systems, this is what dependency('package') is for.

Also, adding a dependency to a test should not mean that you drag a link dependency for the entire library, a pixbuf loader should not depend on GTK unless there is a good reason.

You can have optional tests that only get built if GTK is present but I want those to be independent programs, you can do conditionals (if blocks) using the dependency() return value, have a look at the meson documentation.